tests/terraform/checks/resource/aws/test_LambdaCodeSigningConfigured.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import hcl2
+
+from checkov.terraform.checks.resource.aws.LambdaCodeSigningConfigured import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestLambdaCodeSigningConfigured(unittest.TestCase):
```

Comment:
you used the wrong style for running the tests, please do something similar like here https://github.com/bridgecrewio/checkov/blob/master/tests/terraform/checks/resource/aws/test_WAFACLCVE202144228.py

Reply:
fix please check